test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_seekTime_seconds_Upper_Bound.lua

```diff
@@ -0,0 +1,94 @@
+--------------------------------------------------------------------------------------------
```

Comment:
@OVikhrov, script crashes on start with:
Lua error:
./user_modules/shared_testcases/commonFunctions.lua:330: attempt to perform arithmetic on global 'NewTestSuiteNumber' (a nil value)